README.md

```diff
@@ -71,22 +91,37 @@ For example creating a new customer with a website requires an `intpCustomerId`
 
 ## Available APIs
 
-- [Customers](#customers-api)
```

Comment:
oh btw, i think you need to update the fragment links since you changed the heading text